### PR TITLE
virtio_fs: use "-o xattrmap" without "-o xattr"

### PR DESCRIPTION
**Pull request by xiagao**
ID: 2012735
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
@zhenyzha @YongxueHong Call for your review.thx.

File: qemu/tests/cfg/virtio_fs_set_capability.cfg [hunk 1]
```diff
@@ -97,3 +97,7 @@
                     cmd_set_capability = 'setcap cap_net_raw+eip ${fs_dest}/test_file'
                     cmd_get_capability = 'getcap ${fs_dest}/test_file'
                     cmd_echo_file = 'echo "hello" > ${fs_dest}/test_file'
+                    variants:
+                        - with_xattr:
+                        - without_xattr:
+                            fs_binary_extra_options += ' -o xattr'
```
Comment (YongxueHong) [inline qemu/tests/cfg/virtio_fs_set_capability.cfg:103]:
```suggestion
                        - with_xattr:
                            fs_binary_extra_options += ' -o xattr'
                        - without_xattr:
```
From my understanding and the variant name, should move `fs_binary_extra_options += ' -o xattr'` into the case with_xattr, right?

Comment (xiagao) [reply qemu/tests/cfg/virtio_fs_set_capability.cfg]: You are right,it's my bad. I have updated accordingly. Thanks.

**Review by YongxueHong — APPROVED** [review]

**Review by YongxueHong — APPROVED** [review]

**Comment by zhenyzha** [comment]
```
 (1/3) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_dac_read_search.with_cache.auto.arm64-pci: STARTED
 (1/3) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_dac_read_search.with_cache.auto.arm64-pci: PASS (64.46 s)
 (2/3) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_sys_admin.with_xattr.with_cache.none.arm64-pci: STARTED
 (2/3) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_sys_admin.with_xattr.with_cache.none.arm64-pci: ERROR: Timeout expired while waiting for shell command to complete: 'cat /proc/mounts'    (output: '') (224.16 s)
 (3/3) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_sys_admin.without_xattr.with_cache.none.arm64-pci: STARTED
 (3/3) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.virtio_fs_set_capability.remove_capability.cap_sys_admin.without_xattr.with_cache.none.arm64-pci: ERROR: Timeout expired while waiting for shell command to complete: 'cat /proc/mounts'    (output: '') (224.25 s)
```

The ERROR is since of a product bug

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Review by zhenyzha — APPROVED** [review]

**Pull request merged**